frontend/ui/data/css_tweaks.lua

```diff
@@ -478,6 +478,24 @@ sup { font-size: 50% !important; vertical-align: super !important; }
 sub { font-size: 50% !important; vertical-align: sub !important; }
                 ]],
             },
+            (function()
+                local sub_table = {
+                    title = _("Override normal font line leight"),
+                }
+                for __, rem in ipairs( { 0.9, 0.95, 1.0, 1.05, 1.1, 1.15, 1.2, 1.25, 1.3, 1.35, 1.4, 1.45, 1.5 } ) do
+                    table.insert(sub_table, {
+                        id = T("normal_line-height_%1", rem * 100),
+                        conflicts_with = function(id) return util.stringStartsWith(id, "normal_line-height_") end,
+                        title = T(_("Normal line height: %1"), rem),
+                        css = T([[
+body {
+    -cr-normal-line-height: %1 !important;
+}
+                        ]], rem),
```

Comment:
For small singles ones like this, you can have it on a single line, and no need for !important as it is a private properties no publisher will be using.
`css = T([[body { -cr-normal-line-height: %1 }]], rem),`
(also, `rem` feels like not the right variable name, `num` feels ok)